core/src/main/java/fr/sncf/osrd/railjson/schema/schedule/RJSAllowanceRange.java

```diff
@@ -0,0 +1,21 @@
+package fr.sncf.osrd.railjson.schema.schedule;
+
+import com.squareup.moshi.Json;
+
+public final class RJSAllowanceRange {
```

Comment:
this class lacks a default constructor, which means fields will be initialized to 0 or null
with moshi, all fields are optional. you should initialize these fields to a known default value which is distinguishable from missing, like Double.NaN, and check for that in parseAllowanceRange